### PR TITLE
add local facts uploader docs

### PR DESCRIPTION
**Pull request by omer9564**

### CODE REVIEW

File: docs/how-to/manage-data/local-facts-uploader.mdx [hunk 1]
```diff
@@ -0,0 +1,112 @@
+---
+sidebar_position: 4
+title: Upload facts via the PDP
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+To help reduce the sync latency between Permit cloud and the PDP,
```
Comment (obsd) [inline docs/how-to/manage-data/local-facts-uploader.mdx:9]: To help reduce the data sync latency (the user don't care about implementation)

File: docs/how-to/manage-data/local-facts-uploader.mdx [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+the PDP can be used for some of the APIs.
+You don't have to worry about the unsupported APIs as the SDK takes care of it for you.
```
Comment (obsd) [inline docs/how-to/manage-data/local-facts-uploader.mdx:11]: is it right?

Comment (omer9564) [reply docs/how-to/manage-data/local-facts-uploader.mdx:11]: Yes

**Review by asafc — APPROVED** [review]
overall the information is here, but:
- this document could use some proof-reading / streamlining.

it's super easy to let ChatGPT proof read and streamline the text.
You could also ask @danielbass37 help.

I approve (so you won't be blocked) but please improve this doc.

**Review by obsd — APPROVED** [review]

**Pull request merged**